DeviceAdministration/Web/DataTables/DataTablesResponse.cs

```diff
@@ -1,10 +1,20 @@
-﻿namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Web.DataTables
+﻿using Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Infrastructure.Models;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Web.DataTables
 {
-    public class DataTablesResponse
+    public class RuleDataTablesResponse
```

Comment:
make DataTablesResponse generic or inherit from DataTablesResponse for both Rule and Action Tables